ui/apps/platform/src/Components/PatternFly/CheckboxSelect.tsx

```diff
@@ -1,16 +1,47 @@
-import React, { ReactElement, useState, useRef } from 'react';
+import React, { ReactElement, ReactNode, useState, useRef, useMemo } from 'react';
 import {
     Select,
     SelectOption,
     SelectOptionProps,
+    SelectGroup,
     MenuToggle,
     MenuToggleElement,
     Badge,
     Flex,
     FlexItem,
     SelectList,
+    SelectPopperProps,
 } from '@patternfly/react-core';
 
+// Enhance children to automatically inject hasCheckbox and isSelected props
+function enhanceSelectOptions(children: ReactNode, selectionsSet: Set<string>): ReactNode {
+    return React.Children.map(children, (child) => {
+        if (React.isValidElement(child)) {
+            if (child.type === SelectOption) {
+                const { value } = child.props;
+                if (value != null) {
```

Comment:
👀  I have to imagine `!=` is forbidden by eslint - I don't think I have seen this elsewhere.

Reply:
Oops 👀 Good eye